tests/vxlan/test_scale_ecmp.py

```diff
@@ -0,0 +1,258 @@
+import json
+import time
+import sys
+from ipaddress import IPv4Address
+import pytest
+import logging
+import traceback
+from tests.common.reboot import reboot
+from tests.ptf_runner import ptf_runner
+from tests.common.fixtures.ptfhost_utils import copy_ptftests_directory     # noqa:F401
+from tests.common.vxlan_ecmp_utils import Ecmp_Utils
+from tests.common.helpers.assertions import pytest_assert, pytest_require
+
+logger = logging.getLogger(__name__)
+ecmp_utils = Ecmp_Utils()
+
+pytestmark = [
+    pytest.mark.topology('t0'),
+    pytest.mark.disable_loganalyzer,
+]
+
+VNET_NAME = "Vnet1"
+TUNNEL_NAME = "tunnel_v4"
+VNI = 10000
+PREFIX = "150.0.3.1/32"
+INITIAL_ENDPOINTS = ["100.0.1.10", "100.0.2.10"]
+CHANGED_ENDPOINTS = ["100.0.3.10", "100.0.4.10"]
+PACKET_MULTIPLIER = 10
+
+
+# ---------- Utility ----------
+def get_loopback_ip(cfg_facts):
+    for key in cfg_facts.get("LOOPBACK_INTERFACE", {}):
+        if key.startswith("Loopback0|") and "." in key:
+            return key.split("|")[1].split("/")[0]
+    pytest.fail("Cannot find IPv4 Loopback0 address in LOOPBACK_INTERFACE")
+
+
+def apply_chunk(duthost, payload, name):
+    content = json.dumps(payload, indent=2)
+    dest = f"/tmp/{name}.json"
+    duthost.copy(content=content, dest=dest)
+    duthost.shell(f"sonic-cfggen -j {dest} --write-to-db")
+
+
+def _update_vxlan_endpoints(duthost, vnet, prefix, endpoints, vni):
+    logger.info(f"Updating VNET_ROUTE_TUNNEL {vnet}|{prefix} with {len(endpoints)} endpoints")
+    ep_str = ",".join(endpoints)
+    duthost.shell(
+        f"sonic-db-cli CONFIG_DB hmset 'VNET_ROUTE_TUNNEL|{vnet}|{prefix}' "
+        f"endpoint '{ep_str}' vni '{vni}'"
```

Comment:
VNET_ROUTE_TUNNEL accepts mac_address and vni fields, these are missing from the test, can you add a test case for those 2 fields being tested.